utils/build_all.sh

```diff
@@ -17,4 +17,4 @@ echo 'copying import/export scripts for snapshots'
 cp -R database/*.sh build/database
 cp -R database/*.sql build/database
 echo 'building plugins'
-node ./utils/pluginScripts/buildPlugins.js
+node --unhandled-rejections=strict ./utils/pluginScripts/buildPlugins.js
```

Comment:
Andrei provided this -- it should make the whole docker build stop on any error, to avoid seeing this problem again.

Reply:
I don't think we have time to confirm this before updating our servers, but it'd be good if someone can test it later this week by intentionally making one of the plugins non-compile-able (just change a test), and then running "dockerise" and check that it does fail correctly.